docs/notes-pp.md

```diff
@@ -5,4 +5,56 @@ of the permutation polynomial evaluations.
 Permutation polynomials
 ======================
 
-A permutation
\ No newline at end of file
+A permutation polynomial is bijective 
+function which describes a permutation, 
+of a group of elements. These functions 
+are commonly denoted as polynomial 
```

Comment:
This does not make much sense for me. If a "permutation polynomial is a bijective function". Doesn't that make it a polynomial function? Also, is there a reference to permutation polynomials which permute infinite fields? This sentence seems to imply that that is what a general permutation polynomial is, IIUC